scripts/build_upload.sh

```diff
@@ -96,7 +96,7 @@ echo "Removing first bokeh build at $first_build_loc"
 #########################
 
 # build for each python version
-for py in 27 33 34;
```

Comment:
I don't think so... but if you want to use the 3.4 conda package you just need to make a 3.4 environment... I don't think it is too much work to ask...